Makefile

```diff
@@ -375,10 +381,8 @@ currentversion:
 	#echo $(shell readlink $(CURRENT) | sed -E 's/rootfs-(.*)\.[^.]*$/\1/')
 	@cat $(CURRENT_DIR)/installer/eve_version
 
-
 kernel-yml:
-	$(QUIET)if [ ! -f "pkg/kernel/build.yml" ] || [ ! -f "pkg/new-kernel/build.yml" ]; then tools/kernel-build-yml.sh; fi
-
+	$(QUIET)$(KERNEL_BUILD_YML_TOOL)
```

Comment:
Do I get it right, that now kernel-yml will always be quiet?

Reply:
It seems nothing has been changed, QUIET was always there.

Reply:
I guess I like python more than bash :D But nevertheless, you are removing check if `pkg/kernel/build.yml` and `pkg/new-kernel/build.yml` files exist and instead we always put `tools/kernel-build-yml.sh` with PREEMT_RT variable. Why do we skip it now?

Reply:
The commit message should tell the whole story: https://github.com/lf-edge/eve/pull/3254/commits/e66a85ad6feeb2086aa350435d1e7791ecc0fcd5
What's probably is missing in the description is the following line: "We need to generate the build.yml each time to reflect the changes in the PREEMPT_RT_SUPPORT build argument"